testing/steps/test_output_transformer_wrapper.py

```diff
@@ -39,7 +39,8 @@ def test_output_transformer_should_zip_data_input_and_expected_output_in_the_tra
             current_ids=[0, 1, 2],
             data_inputs=[1, 2, 3],
             expected_outputs=[2, 3, 4]
-        )
+        ),
+        ExecutionContext.from_root(pipeline, tmpdir)
```

Comment:
If I understand, you could have send the root path in the constructor of the pipeline object, and also in the setup of the pipeline object. Am I right? That would be cozy to have multiple opportinuties to set or update the root path before the first transform.

Reply:
yes, you could change the context if you like 👍